test/services/helpers/services/homeworks.js

```diff
@@ -4,6 +4,11 @@ let createdHomeworks = [];
 
 const create = (opt) => async (data) => {
 	data.schoolId = data.schoolId || opt.schoolId;
+	data.name = data.name || 'testHomework';
+	data.availableDate = data.availableDate || new Date();
+	data.teacherId = data.teacherId || opt.generateObjectId();
+	data.private = data.private || false;
```

Comment:
private should default to null as it is a nullable boolean per definition in the model